src/guiapi/src/Jogwheel.cpp

```diff
@@ -60,9 +61,15 @@ void Jogwheel::ReadInput(uint8_t &signals) {
     }
 }
 
-const Jogwheel::ButtonAction Jogwheel::GetButtonAction() {
-    ButtonAction ret = btn_action;
-    btn_action = ButtonAction::BTN_NO_ACTION;
+volatile bool Jogwheel::ConsumeButtonEvent(Jogwheel::BtnState_t &ev) {
+    static uint32_t last_read;          //cannot read it too often
+    const uint32_t min_read_dellay = 8; //ms
+    if ((last_read - HAL_GetTick()) < min_read_dellay) {
```

Comment:
I don't see where is "last_read" assigned to some value. Here on row:65 and on row:117, it seems to me like this variable is uninitialized. Am I wrong?

Reply:
fixed